src/ui.js

```diff
@@ -75,6 +76,7 @@ function getParamsFromUI() {
     };
     //technical params
     params["ethUsdTrendSampleDays"] = parseInt(document.getElementById("ethUsdTrendSampleDays").value);
+    rates.rates.refreshPeriod = parseInt(document.getElementById("refreshPeriod").value);
```

Comment:
as a I suggested earlier, pls move this param to simulation
also pls rename it to graphRefreshPeriod